clients/privacy-center/app/server-environment.ts

```diff
@@ -45,6 +45,7 @@ export interface PrivacyCenterSettings {
   OVERLAY_PARENT_ID: string | null; // (optional) ID of the parent DOM element where the overlay should be inserted
   MODAL_LINK_ID: string | null; // (optional) ID of the DOM element that should trigger the consent modal
   PRIVACY_CENTER_URL: string; // e.g. http://localhost:3000
+  TCF_ENABLED: boolean; // whether we should render the TCF modal
```

Comment:
for explicitness sake, could we set this env var in `fides-js-components-demo.html` where we initialize `window.Fides`?